js/CRM_Admin_Form_Options.js

```diff
@@ -4,16 +4,12 @@ CRM.$(function($) {
   var handleSelectChange = function handleSelectChange(e){
     $('input#value').val(this.value);
     $('input#label').val(this.value);
-  }
+  };
 
   jqEl = $('input#value');
   jqEl
     .hide()
-    .before('\
-      <select class="crm-form-select crm-select2" id="cpvolhours-select-value">\n\
-        <option></option>\n\
-      </select>\n\
-    ');
+    .before('\ <select class="crm-form-select crm-select2" id="cpvolhours-select-value">\n\ <option></option>\n\ </select>\n\ ');
```

Comment:
JSHint is saying this should all be on one line, so that's good. In this case, please go ahead and remove the `\` and `\n` strings so we just have plain HTML markup in this string.

Reply:
Instead of removing the `\` and '\n'. I added the `/* jshint multistr: true */` as suggested on the error.